logging/logmanager/plugin_impl_log_manager.go

```diff
@@ -216,3 +229,88 @@ func stringToLogLevel(level string) logging.LogLevel {
 
 	return logging.InfoLevel
 }
+
+const (
+	HookSysLog   = "syslog"
+	HookLogStash = "logstash"
+	HookFluent   = "fluent"
+)
+
+type levelMapping struct {
+	syslogLevel map[logging.LogLevel]syslog.Priority
+	fluentLevel map[logging.LogLevel][]logrus.Level
+}
+
+type HookConfig struct {
+	Protocol string
+	Address  string
+	Port     int
+	Level    string
```

Comment:
We will want slice here. There should be option to set for example both error and warning for hooks.